packages/compass-crud/package.json

```diff
@@ -34,6 +34,7 @@
     ".": "./src/index.ts"
   },
   "scripts": {
+    "bootstrap": "npm run postcompile",
```

Comment:
I would really prefer us to do something about the need for this before we land adding d.ts compilation to all plugins. It can be done with tsconfig paths option pretty easily which will cover compile and programmatic eslint usage, but needs to be also accounted for VSCode which is not the most straightforward thing to do (it doesn't want to resolve the paths correctly). If this patch is required for something urgent feel free to merge it anyway with the bootstrap added to every package

Reply:
No, it’s not urgent per se. It’s just that we’re starting to be in a position in which we use TS packages in other TS packages and just don’t get proper types in that case, and this PR didn’t run into any major issues with mismatching types, but I’d really like to make sure that that’s not happening in the future.

Reply:
@gribnoysup Alright, I’ll take your :+1: to mean that it’s okay to merge this :) But to be clear, I’m definitely not opposed to changing this in the future if there’s a better way to achieve this goal

Reply:
Yeah, sorry, that definitely means it was okay to merge (I started to write a comment, but got distracted)